src/chainparams.cpp

```diff
@@ -20,7 +20,7 @@
 #include "chainparamsseeds.h"
 
 // FIXME This need to be update every new fork
-uint64_t nMiningForkTime = 1557921600;
+uint64_t nMiningForkTime = 1573819200;
```

Comment:
Nit: How about a `const uint64_t GRAVITON_ACTIVATION_TIME = 1573819200`, rather than repeating this number 4 times in this file?

Reply:
makes sense. having a constant would be better, even thou I'd rather chose another name for it, "gravitation" it is ABC specific.

Reply:
const seems fine but what's wrong with nMiningForkTime...IMO seems more intuitive than some other name.

Reply:
@ptschip `nMiningForkTime` is used in the definition of the `miningForkTime` so it can't be `const`. Having said that in commit  
9bc2e24 I used this var to initliaze all chain params related to activation time

Reply:
Use NOV2019_ACTIVATION_TIME.  Using a code name here makes no sense, and may be confusing in the future (if we don't remove the constant post upgrade) since it will be hard to remember what code name goes to what.